src/components/Button/ButtonBase.js

```diff
@@ -17,7 +17,7 @@ export const ButtonBase = styled.button`
   height: 40px;
   font-size: ${(props) => props.theme.typography.bodyFontSizes[0]};
   line-height: ${(props) => props.theme.typography.bodyLineHeights[0]};
-  padding: 8px 20px;
+  padding: 0 2.5rem;
```

Comment:
The correct horizontal padding according to the design system is 20px, did you make this change on purpose?
I'm using [this](https://xd.adobe.com/spec/1ba426af-54fd-45a2-6d8c-9d5508ab2df0-6d95/screen/a03d45f6-ae14-4d2b-9eba-13e9a24dba7d/Botones/) as a reference. Am I looking at the correct spec?

Reply:
You're right. I've tried some things and I forgot to revert that.